modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -50,45 +50,51 @@
       </AposModalRail>
     </template>
     <template #main>
-      <AposModalBody>
+      <AposLoadingBlock v-if="isFirstLoading" />
+      <AposModalBody v-else>
         <template #bodyHeader>
           <AposDocsManagerToolbar
             :selected-state="selectAllState"
             :total-pages="totalPages"
             :current-page="currentPage"
             :filters="toolbarFilters"
+            :filter-values="filterValues"
             :labels="moduleLabels"
             :disable="relationshipErrors === 'min'"
             :displayed-items="items.length"
             :checked="checked"
             :checked-count="checked.length"
             :module-name="moduleName"
-            @page-change="updatePage"
+            :options="{noPager: true}"
             @select-click="selectClick"
             @search="search"
             @filter="filter"
           />
         </template>
         <template #bodyMain>
+          <AposLoadingBlock v-if="isLoading" />
           <AposMediaManagerDisplay
+            v-else
             ref="display"
             :checked="checked"
             :accept="accept"
             :items="items"
             :module-options="moduleOptions"
             :max-reached="maxReached()"
+            :is-last-page="totalPages > 1 && currentPage === totalPages"
```

Comment:
what happen when I have only 1 page?

Reply:
We do not show `you've reached the end of media manager` message.